dev-packages/e2e-tests/test-applications/sveltekit/test/performance.server.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '../event-proxy-server';
+
+test('server pageload request span has nested request span for sub request', async ({ page }) => {
+  const serverTxnEventPromise = waitForTransaction('sveltekit', txnEvent => {
+    return txnEvent?.transaction === 'GET /server-load-fetch';
+  });
+
+  await page.goto('/server-load-fetch');
+
+  const serverTxnEvent = await serverTxnEventPromise;
+  const spans = serverTxnEvent.spans;
+
+  expect(serverTxnEvent).toMatchObject({
+    transaction: 'GET /server-load-fetch',
+    tags: { runtime: 'node' },
+    transaction_info: { source: 'route' },
+    type: 'transaction',
+    contexts: {
+      trace: {
+        op: 'http.server',
+        origin: 'auto.http.sveltekit',
+      },
+    },
+  });
+
+  expect(spans).toHaveLength(2);
```

Comment:
l: I'd argue that this assertion is redundant since we already have the one checking for the two relevant spans below. Just pointing out because it may create unnecessary test failures in case there is ever gonna be additional spans because of instrumentation changes. Same for the sveltekit 2 test!